moto/cognitoidentity/responses.py

```diff
@@ -34,10 +36,10 @@ def update_identity_pool(self):
         allow_classic = self._get_bool_param("AllowClassicFlow")
         login_providers = self._get_param("SupportedLoginProviders")
         provider_name = self._get_param("DeveloperProviderName")
-        provider_arns = self._get_multi_param("OpenIdConnectProviderARNs")
```

Comment:
Looks like we were using the wrong methods here - Cognito Identity is already using JSON payloads (and not URL-encoded form data), hence we can simply use `_get_param(..)` here.